docs/api/aave_v2_stablecoin.md

```diff
@@ -0,0 +1,199 @@
+# Protocol Documentation
+<a name="top"></a>
+
+## Table of Contents
+
+- [aave_v2_stablecoin.proto](#aave_v2_stablecoin-proto)
+    - [AaveV2Stablecoin](#steward-v1-AaveV2Stablecoin)
+    - [AaveV2Stablecoin.AccrueFees](#steward-v1-AaveV2Stablecoin-AccrueFees)
+    - [AaveV2Stablecoin.ClaimAndUnstake](#steward-v1-AaveV2Stablecoin-ClaimAndUnstake)
+    - [AaveV2Stablecoin.EnterPosition](#steward-v1-AaveV2Stablecoin-EnterPosition)
+    - [AaveV2Stablecoin.Rebalance](#steward-v1-AaveV2Stablecoin-Rebalance)
+    - [AaveV2Stablecoin.Rebalance.SwapParams](#steward-v1-AaveV2Stablecoin-Rebalance-SwapParams)
+    - [AaveV2Stablecoin.Reinvest](#steward-v1-AaveV2Stablecoin-Reinvest)
+    - [AaveV2Stablecoin.Sweep](#steward-v1-AaveV2Stablecoin-Sweep)
+    - [AaveV2Stablecoin.TransferFees](#steward-v1-AaveV2Stablecoin-TransferFees)
+  
+- [Scalar Value Types](#scalar-value-types)
+
+
+
+<a name="aave_v2_stablecoin-proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## aave_v2_stablecoin.proto
+
+
+
+<a name="steward-v1-AaveV2Stablecoin"></a>
+
+### AaveV2Stablecoin
+Represents a function call to the Aave V2 Stablecoin cellar
+
+
+| Field | Type | Label | Description |
```

Comment:
If the `label` and `description` fields are going to be empty, it'll be best to leave them out.

Reply:
> We shouldn't muck with the autogenerated files unless there's a way to adjust the generator's configuration.

added comments so it would generate descriptions